tijanabekic/assignment1/Assignment 1.java

```diff
@@ -0,0 +1,160 @@
+Q1. Given two strings, determine if one is an anagram of the other. Two words are anagrams of

```

Comment:
Please make sure these comments are in comments and put the code within a class.

```
/**
 * Given two strings, determine if one is an anagram of the other. Two words are anagrams of
 * each other if they are made of the same letters in a different order
 */
class Question1 {
  boolean isAnagram(String si, String s2) {
    ...
  }
  ...
}
```